nsxt/resource_nsxt_policy_ods_runbook_invocation.go

```diff
@@ -0,0 +1,202 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/infra/sha"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicyODSRunbookInvocation() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyODSRunbookInvocationCreate,
+		Read:   resourceNsxtPolicyODSRunbookInvocationRead,
+		Update: resourceNsxtPolicyODSRunbookInvocationUpdate,
+		Delete: resourceNsxtPolicyODSRunbookInvocationDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtPolicyPathResourceImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id": getNsxIDSchema(),
+			"path":   getPathSchema(),
+			// Due to a bug, invocations with a display_name specification fail, and when there's none set, NSX assigns
```

Comment:
So the user can't specify display_name?

Reply:
For now, he can't. The runbook won't execute, it's a bug which they intend to fix obviously.